File: Daily Challanges/2026-04-April/2026-04-20.py
DICTIONARY = {
    "NASA": "National Avocado Storage Authority",
    "CIA": "Cats Infiltration Agency",
    "FBI": "Fluffy Beanbag Inspectors",
    "DOJ": "Department Of Jelly",
    "WHO": "Wild Honey Organization",
    "EPA": "Eating Pancakes Administration",
}


def find_org(acronym):
    return DICTIONARY[acronym]


print(find_org("NASA"))
print(find_org("CIA"))
print(find_org("FBI"))

File: Daily Challanges/2026-04-April/2026-04-20.js
const DICT = {
  NASA: "National Avocado Storage Authority",
  CIA: "Cats Infiltration Agency",
  FBI: "Fluffy Beanbag Inspectors",
  DOJ: "Department Of Jelly",
  WHO: "Wild Honey Organization",
  EPA: "Eating Pancakes Administration",
};

function findOrg(acronym) {
  return DICT[acronym];
}

console.log(findOrg("NASA"));
console.log(findOrg("CIA"));
console.log(findOrg("FBI"));
